src/commands/classes/_common.ts

```diff
@@ -0,0 +1,89 @@
+import ExecutionContext from '../../ExecutionContext';
+import * as Discord from '../../Discord';
+
+export type dbClass = {
+  name: string;
+  description: string;
+  id: string;
+  members: string[];
+};
+
+export const classExists = async (
+  ctx: ExecutionContext,
+  guildId: string,
+  name: string,
+): Promise<boolean> => {
+  const db = await ctx.db.getDb(guildId);
+
+  const classObj = await db.collection('classes').findOne({ name: name });
```

Comment:
Implement @aiyan's suggestion (https://github.com/nu25/hoosky-bot/pull/11#discussion_r653172031) and lookup classes by CRN instead of name

```suggestion
  const classObj = await db.collection(Collection.CLASSES).findOne({ crn });
```